data/korath/korath missions.txt

```diff
@@ -257,3 +257,598 @@ mission "Small Scale Delivery"
+					accept
+				`	"On second thought, I don't think I want this mission. Illegal fish deliveries are too strange for my tastes."`
+					decline
+				`	"If I'm just delivering fish, why is it illegal?"`
+					to display
+						not "Korath [F01]: A Fishy Deal: spicy"
+			label "'cause they're spicy"
+			action
+				set "Korath [F01]: A Fishy Deal: spicy"
+			`	"Nothing is wrong with the fish; the other end of the delivery is the problem, so the whole trade arrangement is illegal. There's all sorts of barrels and boxes that come back from <planet>. It's mostly mundane stuff; never any drugs, counterfeits, or weapons. Why the Republic would ban this... I haven't the slightest notion.`
+			`	"A while back, my aunt was fined 25,000 credits for using a knock-off fuel gasket on her racing Flivver. She used the gasket because it was more reliable than a genuine one. Of course, that doesn't matter to the Republic. The Navy officer that caught her knew how dumb the law was, so he let her keep the gasket."`
+				goto "d'ya want the job"
+
+	on enter Atik
+		dialog "It seems you do not have automatic clearance to <planet>. If you talk to the planet, the authorities will probably let you land."
+	on visit
+		dialog "You've arrived at <planet> but you left some of the fish behind. You should take off and wait for your escorts to arrive."
+
+event "know about Korath fish jobs"
+
+
+
+mission "Korath [F02]: A Fishy Proposition"
+	landing
+	name "A fishy proposition"
+	description "Transport <cargo> to <planet> by <date> for <payment>. Avoid getting scanned or landing on high-security planets, or you may be fined."
+	cargo "planetary supplies" 15
+	# See note above in "Korath [F01]: A Fishy Deal" for why this is illegal.
+	illegal 50000
+	"apparent payment" 140000
+	deadline 4
+	clearance "The authorities do not respond, but a concealed hangar bay door opens. The meaning is clear: land here."
+	source "La'ab Bamen"
+	destination "New Tortuga"
+	to offer
+		has "Korath [F01]: A Fishy Deal: done"
+	on offer
+		conversation
+			`You approach the asteroid and a crater suddenly opens up, revealing a docking bay. Entire human space stations could fit in here, but there are only a few ordinary ships parked. When you exit the <ship>, several dire men in uniformed body armor greet you. Each holds a bright orange rifle whose scorching heat warps the air around them. They all stare intensely at you as another figure steps forward.`
+			`	It is a woman who captures your attention, with a warm and open smile on her face. Strapped over the top of her silky white dress you see two revolvers and a sizable supply of ammo. Beyond that there is a knife on each leg, just above her knee-high combat boots, and you think there's another one between her shoulder-blades. A rainbow lock contrasts her otherwise light blond hair, and matches the rainbow feather on a necklace draped around her neck.`
```

Comment:
```suggestion
			`	It is a woman who captures your attention, with a warm and open smile on her face. Strapped over the top of her silky white dress you see two revolvers and a sizable supply of ammo. Beyond that there is a knife on each leg, just above her knee-high combat boots, and you think there's another one between her shoulder blades. A rainbow lock contrasts her otherwise light blond hair, and matches the rainbow feather on a necklace draped around her neck.`
```